lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+module Players
+
+  class Human < Player
+
+    def move(input)
+      puts "It's your turn #{token}! What's your move? (Enter a number between 1 and 9)"
+      input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "It's your turn #{token}! What's your move? (Enter a number between 1 and 9)"
  gets.strip
end
```